js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ka.properties

```
@@ -90,7 +90,7 @@ passwordless=უპაროლო
 sharedWithMe=ჩემთან გაზიარებული
 recovery-authn-code=ჩემი აღდგენის ავთენტიკაციის კოდები
 webauthn-help-text=გამოიყენეთ თქვენი Passkey შესასვლელად.
-error-invalid-multivalued-size=ატრიბუტს {{0}} უნდა ჰქონდეს მინიმუმ {{1}} და მაქსიმუმt {{2}} მნიშვნელობა.
+error-invalid-multivalued-size=ატრიბუტს {{0}} უნდა ჰქონდეს მინიმუმ {1} და მაქსიმუმt {2} მნიშვნელობა {2,choice,0#values|1#value|1<values}.
```

@NorwayFun - I see that you took the original text for `{2,choice,0#values|1#value|1<values}` without changes.

The `choice` format you see here is intended to allow different forms for singular or plural. In English this allows to print "0 values, 1 value, 2 values, ...". I don't know about the Georgian language. If there is only one form, you can simplify it as

```suggestion
error-invalid-multivalued-size=ატრიბუტს {0} უნდა ჰქონდეს მინიმუმ {1} და მაქსიმუმt {2} {2,choice,0#მნიშვნელობა,0<მნიშვნელობა}.
```

---

Thanks!